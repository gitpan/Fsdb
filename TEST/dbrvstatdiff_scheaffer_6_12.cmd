# example 6.12 from Scheaffer and McClave
prog='dbrvstatdiff'
args='mean2 stddev2 n2 mean1 stddev1 n1'
cmp='diff -cb '
altcmp='dbfilediff -E --exit '
altcmp_needs_input_flags=true
