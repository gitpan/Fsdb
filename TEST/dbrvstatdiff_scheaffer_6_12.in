#h title mean2 stddev2 n2 mean1 stddev1 n1
example6.12 0.17 0.0020 5 0.22 0.0010 4
