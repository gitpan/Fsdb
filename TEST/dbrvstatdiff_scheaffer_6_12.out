#fsdb title mean2 stddev2 n2 mean1 stddev1 n1 diff diff_pct diff_conf_half diff_conf_low diff_conf_high diff_conf_pct_half diff_conf_pct_low diff_conf_pct_high
example6.12	0.17	0.0020	5	0.22	0.0010	4	0.05	29.412	0.0026138	0.047386	0.052614	1.5375	27.874	30.949
#   | dbrvstatdiff mean2 stddev2 n2 mean1 stddev1 n1
